org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/EclipseSbtBuildManager.scala

```diff
@@ -186,7 +185,7 @@ class EclipseSbtBuildManager(val project: IScalaProject, settings: Settings, ana
 
   // take by-name argument because we need incOptions only when we have a cache miss
   override def latestAnalysis(incOptions: => IncOptions): Analysis =
```

Comment:
Unfortunately `CompileAnalysis` is just a marker interface, we can't do anything with it.